package.json

```diff
@@ -52,6 +53,11 @@
   "resolutions": {
     "csstype": "^2.6.18"
   },
+  "overrides": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
these react overrides are for the benefit of `react-beautiful-dnd` (a drag-and-drop library). This is no longer actively maintained, so we should consider removing/replacing it

Reply:
Earlier on when we were looking at this I was translating `dnd` => dungeons and dragons 😄